spring-data-sample/README.md

```diff
@@ -0,0 +1,262 @@
+# Spring Data integration Sample
+
+This is a sample Spring Boot application for Spring Data integration with ScalarDB.
+
+## Prerequisites
+- Java (OpenJDK 8 or higher)
+- Gradle
+- Docker, Docker Compose
+
+## Sample application
+
+### Overview
+
+This article describes how to create a Spring Boot applicaiton for the same use case as [ScalarDB Sample](https://github.com/scalar-labs/scalardb-samples/tree/main/scalardb-sample) using Spring Data integration with ScalarDB.
+Please note that application-specific error handling, authentication processing, etc., are omitted in the sample application since it focuses on explaining how to use Spring Data integration.
```

Comment:
```suggestion
Please note that application-specific error handling, authentication processing, etc. are omitted in the sample application since this tutorial focuses on explaining how to use Spring Data integration.
```